CHANGELOG.md

```diff
@@ -1,3 +1,255 @@
+
+#### Security enhancements
+
+- `calico-host-protection`: Add custom locked down PSP configuration ([#1274](https://github.com/kinvolk/lokomotive/pull/1274)).
+
+#### Documentation
+
+- Add `openebs-operator` update guide ([#1163](https://github.com/kinvolk/lokomotive/pull/1163)).
+- Add `rook-ceph` update guide ([#1165](https://github.com/kinvolk/lokomotive/pull/1165)).
+
+#### Miscellaneous
+
+- Pull control plane images from Quay to avoid hitting Docker Hub pulling limits ([#1226](https://github.com/kinvolk/lokomotive/pull/1226)).
+- Bootkube now waits for all control plane charts to converge before exiting, which should make the bootstrapping process more stable ([#1085](https://github.com/kinvolk/lokomotive/pull/1085)).
+- Remove deprecated CoreOS mentions from AWS ([#1245](https://github.com/kinvolk/lokomotive/pull/1245)) and bare metal ([#1246](https://github.com/kinvolk/lokomotive/pull/1246)).
+- Improve hardware reservations validation rules on Equinix Metal ([#1186](https://github.com/kinvolk/lokomotive/pull/1186)).
+
+### Updating from v0.5.0
+
+#### Configuration syntax changes
+
+##### AWS
+
+Removed the undocumented `cluster.os_name` parameter, since Lokomotive supports Flatcar Container Linux only.
+
+##### Bare-metal
+
+The `cluster.os_channel` parameter got simplified by removing the `flatcar-` prefix.
+
+###### Old
+
+```hcl
+os_channel = "flatcar-stable"
+```
+
+###### New
+
+```hcl
+os_channel = "stable"
+```
+
+##### Velero
+
+Velero requires an explicit `provider` field to select the provider.
+Example:
+
+```hcl
+component `velero` {
+  provider = "openebs"
+
+  openebs {
+    ...
+  }
+}
+```
+
+#### Cluster update steps
+
+> **NOTE**: Updating multiple Lokomotive versions at a time is not supported. If your cluster is running a version older than `v0.5.0`, update to `v0.5.0` first and only then proceed with the update to `v0.6.0`.
+
+Please perform the following manual steps in your cluster configuration directory.
+
+1. Download the release bundle.
+
+  ```bash
+  curl -LO https://github.com/kinvolk/lokomotive/archive/v0.6.0.tar.gz
+  tar -xvzf v0.6.0.tar.gz
+  ```
+
+2. Install the Packet CCM.
+
+  If you are running Lokomotive on Equinix Metal (formerly Packet), then install Packet CCM. Export your Packet cluster's project ID and API Key.
+
+  ```bash
+  export PACKET_API_KEY=""
+  export PACKET_PROJECT_ID=""
+
+  echo "apiKey: $PACKET_API_KEY
+projectID: $PACKET_PROJECT_ID" > /tmp/ccm-values.yaml
+
+  helm install packet-ccm --namespace kube-system --values=/tmp/ccm-values.yaml ./lokomotive-0.6.0/assets/charts/control-plane/packet-ccm/
+  ```
+
+3. Update all nodes.
+
+  This script shipped with release tarball will update all the nodes to run the latest kubelet, latest etcd and add permanent MetalLB labels.
```

Comment:
Since this updates to the latest kubelet for the non self-hosted case I wonder if we should flip the `--upgrade-kubelets` flag to being `true` by default, it seems weird that we update kubelets but we don't update the self-hosted kubelet...

Reply:
It can fail with issue like this: https://github.com/kinvolk/lokomotive/issues/1313. So I don't know if we should set that flag by default.

Reply:
Fair. We should just fix #1313 though but we can do that after the release. Should we mention something in the update instructions so users pass `--upgrade-kubelets` if they are using self hosted kubelet?